rel/emqx_conf.template.en.md

```diff
@@ -9,18 +9,18 @@ From bottom up:
 
 1. Immutable base: `emqx.conf` + `EMQX_` prefixed environment variables.<br/>
    Changes in this layer require a full node restart to take effect.
-1. Cluster overrides: `$EMQX_NODE__DATA_DIR/configs/cluster-override.conf`
+2. Cluster overrides: `$EMQX_NODE__DATA_DIR/configs/cluster.hocon`
```

Comment:
wrong order, should be:

1. Cluster configs `$EMQX_NODE__DATA_DIR/configs/cluster.hocon`.
2. `emqx.conf` + `EMQX_` prefixed environment variables.